tt_metal/impl/device/device.cpp

```diff
@@ -263,19 +264,23 @@ void Device::initialize_firmware(CoreCoord phys_core, launch_msg_t *launch_msg)
         auto active_eth_cores = this->get_active_ethernet_cores();
 
         if (active_eth_cores.find(logical_core_from_ethernet_core(phys_core)) != active_eth_cores.end()) {
-            int eriscv_id = build_processor_type_to_index(JitBuildProcessorType::ETHERNET).first + 0;
-            ll_api::memory binary_mem = llrt::get_risc_binary(firmware_build_states_[eriscv_id]->get_target_out_path(""));
-            uint32_t kernel_size16 = llrt::get_binary_code_size16(binary_mem, eriscv_id);
-            log_debug(LogDevice, "ERISC fw binary size: {} in bytes", kernel_size16 * 16);
-            llrt::test_load_write_read_risc_binary(binary_mem, this->id(), phys_core, eriscv_id);
+            if (not getenv("TT_METAL_GALAXY_SKIP_LOADING_FW")) {
```

Comment:
is this specific galaxy? If not, we could rename it `TT_METAL_SKIP_LOADING_FW`

Reply:
That makes sense to me. I believe this is not the safest env flag to enable, so left it tagged with GALAXY, since there isn't really an urgent need for this type of feature on non-Galaxy systems. But I can make the change